src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box, Button as ButtonMui, useTheme } from '@mui/material';
+import { h } from 'preact';
+
+import { useOnSubmit } from '../../hooks';
+import {
+  ClickHandler,
+  LaunchAuthenticatorButtonElement,
+  UISchemaElementComponent,
+} from '../../types';
+import { getTranslation } from '../../util';
+import { OktaVerifyIcon } from '../Icon';
+
+const LaunchAuthenticatorButton: UISchemaElementComponent<{
+  uischema: LaunchAuthenticatorButtonElement
+}> = ({ uischema }) => {
+  const theme = useTheme();
+
+  const onSubmitHandler = useOnSubmit();
+  const {
+    translations = [],
+    options: {
+      step,
+    },
+  } = uischema;
+
+  const handleClick: ClickHandler = async () => {
+    onSubmitHandler({
+      step,
+    });
+  };
+
+  const label = getTranslation(translations, 'label');
+  const iconDescription = getTranslation(translations, 'icon-description') || '';
+
+  return (
+    <ButtonMui
+      variant="secondary"
+      fullWidth
+      onClick={handleClick}
+    >
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        alignContent="space-between"
+        gap="5px"
+        // keep the icon from stretching the button vertically
+        marginTop="-4px"
+        marginBottom="-5px"
+      >
+        <OktaVerifyIcon
```

Comment:
Is it possible to use MUI's `startIcon` prop for icon placement on the button?

```
<ButtonMui
      variant="secondary"
      fullWidth
      startIcon={<DeleteIcon />}
      onClick={handleClick}
    >
    ...
```
See: https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

Reply:
This may also alleviate the need for the `<Box` component within the button since MUI controls the icon placement.

Reply:
I did try using the `startIcon` prop but it didn't layout the button properly (placed the icon and text on two lines). It looks like this is something to do with the Odyssey theme, since the issue isn't present if I remove the theme entirely. I will followup with Odyssey to see if this is a bug, but in the meantime will leave this as-is until we have a resolution.

Reply:
Makes total sense! Ty!